### PR TITLE
OKTA-513045 API and OIE Release Notes

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updates to API and OIE API release notes for weekly release; includes minor text and image update to Overview page.
- **Is this PR related to a Monolith release?** 2022.07.1

### Resolves:

* [OKTA-513045](https://oktainc.atlassian.net/browse/OKTA-513045)
* [OKTA-423531](https://oktainc.atlassian.net/browse/OKTA-423531)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/index.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ The release notes are organized by year in descending order. In the left-side na
 
 We release first to preview orgs and then production orgs. Dates for preview releases are the earliest possible release date. Always check your org to verify the release for your org.
 
-To verify the current release for an org, check the footer of the Admin Console. If necessary, click **Admin** to navigate to your Admin Console.<br>
+To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs. If necessary, click **Admin** to navigate to your Admin Console. <br>
+
 ![Release Number in Footer](/img/release_notes/version_footer.png)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/index.md:18]: I don't know if you want to center the image with the new image classes?
`<div class="half"> Release Number in Footer </div>`
the problem with these classes is that "half" is too small, and "three-quarters" is too big. we need an "orig" for original size ;-)

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 2]
```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
...
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bugs fixed in 2022.07.1](#bugs-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set `issuerMode` to `DYNAMIC`, Okta uses the domain from the `authorizeURI` as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
+
+#### Bugs fixed in 2022.07.1
+
+- When the `/api/v1/users/{userId}/roles` or
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:22]:
```suggestion
- When the `/api/v1/users/${userId}/roles` or
```

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 3]
```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
...
+`/api/v1/groups/{groupId}/roles` were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:23]:
```suggestion
`/api/v1/groups/${groupId}/roles` endpoints were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 4]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bug fixed in 2022.07.1](#bug-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set `issuerMode` to `DYNAMIC`, Okta uses the domain from the `authorizeURI` as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]:
```suggestion
You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set  the `issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the `authorizeURI` property as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Where is `authorizeURI` property coming from? (which object?) I don't see this property in the IdP API. Perhaps we need to add references to this property?
Also, is PR 3469 going out with this release?

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 5]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+
+#### Bug fixed in 2022.07.1
+
+When the `/api/v1/users/{userId}/roles` or
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:20]:
```suggestion
When the `/api/v1/users/${userId}/roles` or
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 6]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+`/api/v1/groups/{groupId}/roles` were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:21]:
```suggestion
`/api/v1/groups/${groupId}/roles` endpoints were called to assign a custom role and resource set to a user or group, and those assignments already existed, the calls didn’t receive an HTTP 409 Conflict error. (OKTA-507683)
```
Not sure if we are still using `${var}` notation?

**Review by vanngo-okta — COMMENTED** [review]
A few small nits and a question about the dynamic issuer mode blurb.

File: packages/@okta/vuepress-site/docs/release-notes/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/index.md:18]: Will center original with our workaround.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:22]: Thanks. Updated locally.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md (hunk 3 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:23]: Updated locally.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md (hunk 4 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]: Updated your other suggestions locally.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md (hunk 5 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md:20]: Updated locally.

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md (hunk 6 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2022/index.md:21]: Updated locally. Yes, we're still using the default `${var}` notation for now. No response from Dan's dev survey as far as I know.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 7]
```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
...
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bugs fixed in 2022.07.1](#bugs-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:18]:
```suggestion
You can configure the dynamic issuer mode for an Identity Provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Sorry, I missed this earlier. Our capitalization guide says to capitalize Identity Provider.

File: packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md [hunk 8]
```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2022
...
 ## July
 
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
+| [Bugs fixed in 2022.07.1](#bugs-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md:16]:
```suggestion
#### Dynamic issuer mode for Identity Providers
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 9]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
 
 ## July
 
+### Weekly release 2022.07.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Dynamic issuer mode for identity providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:11]:
```suggestion
| [Dynamic issuer mode for Identity Providers](#dynamic-issuer-mode-for-identity-providers)                    | July 13, 2022            |
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 10]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+| [Bug fixed in 2022.07.1](#bug-fixed-in-2022-07-1)                        | July 13, 2022            |
+
+#### Dynamic issuer mode for identity providers
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:14]:
```suggestion
#### Dynamic issuer mode for Identity Providers
```

File: packages/@okta/vuepress-site/docs/release-notes/2022/index.md [hunk 11]
```diff
@@ -4,6 +4,22 @@ title: Okta API Products release notes 2022
...
+
+You can configure the dynamic issuer mode for an identity provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2022/index.md:16]:
```suggestion
You can configure the dynamic issuer mode for an Identity Provider using the Identity Provider API. When you set the`issuerMode` parameter to `DYNAMIC`, Okta uses the domain from the Authorize URL as the domain for the redirect URI when returning the authentication response. <!--OKTA-506807-->
```

File: packages/@okta/vuepress-site/docs/release-notes/index.md [hunk 12]
```diff
@@ -13,9 +13,14 @@ The release notes are organized by year in descending order. In the left-side na
 
 We release first to preview orgs and then production orgs. Dates for preview releases are the earliest possible release date. Always check your org to verify the release for your org.
 
-To verify the current release for an org, check the footer of the Admin Console. If necessary, click **Admin** to navigate to your Admin Console.<br>
+To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs. If necessary, click **Admin** to navigate to your Admin Console. <br>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/index.md:16]:
```suggestion
To verify the current release and Okta solution for an org (Identity Engine or Classic), check the footer of the Admin Console. The version number is appended with E for Identity Engine orgs and C for Classic Engine orgs.<br>
```
Sorry, I forgot to ask what does the last line refer to? I'm not sure where to click **Admin**? Maybe this last line is old?

Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/index.md:16]: As per Slack convo, keeping it in for now.

**Pull request merged**